### PR TITLE
Fix browser runtime

### PR DESCRIPTION
**Pull request by ospencer**
Looks like I broke the runtime in the browser when I added the WASI stuff, but this fixes that. I also updated the `defaultURLLocator`, since we haven't updated that since we started doing separate compilation.

### CODE REVIEW

**Comment by phated** [comment]
I wonder if we can add puppeteer in CI to verify this.

**Comment by ospencer** [comment]
We could! Though I figure we'd want to do that once we had a better bundle for the stdlib. Could do it sooner though.

File: runtime/src/core/grain-module.js [hunk 1]
```diff
@@ -1,17 +1,27 @@
 import { GrainError } from '../errors/errors';
-import { grainToJSVal } from '../utils/utils';
 
 import { WASI } from "@wasmer/wasi/lib/index.cjs";
-import wasiBindings from "@wasmer/wasi/lib/bindings/node";
+import { WasmFs } from "@wasmer/wasmfs";
+
+const wasmFs = new WasmFs();
```
Comment (phated) [inline runtime/src/core/grain-module.js:6]: Should this be constructed only is `__RUNTIME_BROWSER`

File: runtime/src/utils/locators.js [hunk 2]
```diff
@@ -7,15 +7,20 @@ function normalizeSlash(s) {
...
   return async (raw) => {
-    if (base === null) {
-      return null;
-    }
     let module = raw.replace(/^GRAIN\$MODULE\$/, '');
-    return readURL(base + "/" + module + ".wasm");
+    for (const base of bases) {
+      let fullpath = base + "/" + module + ".wasm";
+      try {
+        return await readURL(fullpath);
```
Comment (phated) [inline runtime/src/utils/locators.js:18]: Not an async function

Comment (ospencer) [reply runtime/src/utils/locators.js:18]: Awkward.

Comment (ospencer) [reply runtime/src/utils/locators.js:18]: Oh yes it is: https://github.com/grain-lang/grain/blob/master/runtime/src/core/grain-module.js#L189

Comment (phated) [reply runtime/src/utils/locators.js:18]: Sorry, I meant the function that you are using the `await` inside, which shouldn't work if it isn't `async`, right?

Comment (ospencer) [reply runtime/src/utils/locators.js:18]: There's an `async` on line 13. A little hard to see.

Comment (phated) [reply runtime/src/utils/locators.js:18]: Ah, didn't realize this function returned another function.

**Pull request merged**